subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/DefaultTransformedFileCache.java

```diff
@@ -161,6 +180,29 @@ private CacheKey getCacheKey(File inputFile, HashCode inputsHash) {
         return new CacheKey(inputFileSnapshot, inputsHash);
     }
 
+    @SuppressWarnings("Since15")
+    private void markAccessedByTouchingTopmostOutputDirectory(List<File> files) {
+        if (files.isEmpty()) {
+            return;
+        }
+        Path baseDir = filesOutputDirectory.toPath();
+        // all files share the same output directory so only taking into account the first one
```

Comment:
⭕️ I prefer well-named extracted methods over inline comments, as the latter go stale quickly.

Reply:
Agreed, that comment was already removed by a subsequent commit.